### ISSUE TITLE
Fix FP S4721: Ignore static template literals

### ISSUE THREAD

**Issue by egon-okerman-sonarsource** [SonarSource/SonarJS#4038]
## Description
Hello JS squad!

During some PoC work I did earlier this year, I came across a small bug in S4721. I have no data about it appearing in community projects, but considering the fix is small I thought it might be worth picking up.

The code for S4721 correctly filters out literals in [`isQuestionable`](https://github.com/SonarSource/SonarJS/blob/f5827a9a2f688354c487e2a4704396eefccded5f/src/linting/eslint/rules/os-command.ts#L62-L72), but unfortunately template literals without any variables are not considered literals. (I saw that you have an [`isStaticTemplateLiteral`](https://github.com/SonarSource/SonarJS/blob/f5827a9a2f688354c487e2a4704396eefccded5f/src/linting/eslint/rules/helpers/ast.ts#L596-L600) helper function which should solve this.)

## Code example
```js
import { execSync } from "child_process";

execSync(`echo "this is fine"`); // FP: currently raises S4721 but should not
```
